doc/kubernetes/modules/ROOT/pages/util/custom-image-for-keycloak.adoc

```diff
@@ -0,0 +1,64 @@
+= Using a custom Keycloak image for deployment in Kubernetes
+:navtitle: Using a custom Keycloak image
+:description: Use a custom image an external registry or a locally built image and run this in Kubernetes.
+
+{description}
+A running version of xref:installation.adoc[Keycloak within Minikube] is a prerequisite.
+
+== Overview
+
+Depending on the operator being used, it will pull a default image from the Keycloak's Quay.io registry.
+When running the nightly operator of Keycloak, it uses the nightly build of Keycloak.
+A specific version of the operator like, for example, 20.0.0, will use the version 20.0.0 of Keycloak.
+
+== Use a custom image available in a remote registry
+
+To change this, edit the environment file `.env` to contain the following:
+
+.Example entry in the `.env` file
+----
+KC_KEYCLOAK_IMAGE=quay.io/keycloak/keycloak:20.0.1
+----
+
```

Comment:
we might want to add a step to specify the user to run `task` after updating their `.env` varibale with a different value

Reply:
also this fails with the custom image with the value `KC_KEYCLOAK_IMAGE=quay.io/keycloak/keycloak:20.0.1` with the below incorrect JDBC url construct, while trying to aquire a JDBC connection. The below error is found on the Keycloak server log, when I ran `task` after updating the env variable with the image

```
{"timestamp":"2022-11-16T16:00:14.624Z","sequence":8439,"loggerClassName":"org.jboss.logging.Logger","loggerName":"io.agroal.pool","level":"WARN","message":"Datasource '<default>': URL format error; must be \"jdbc:h2:{ {.|mem:}[name] | [file:]fileName | {tcp|ssl}:[//]server[:port][,server2[:port]]/name }[;key=value...]\" but is \"jdbc:postgresql://postgres:5432/keycloak\" [90046-214]","threadName":"agroal-11","threadId":21,"mdc":{},"ndc":"","hostName":"keycloak-0","processName":"QuarkusEntryPoint","processId":1}

```

Reply:
Let us have a look at your setup together; I'm wondering if there's been a mix of configurations testing the cockroach-operator setup.